site/docs/hive.md

```diff
@@ -72,6 +72,15 @@ catalog.createTable(tableId, schema, spec, tableProperties);
 
 The table level configuration overrides the global Hadoop configuration.
 
+#### Hive on Tez configuration
+
+To use Tez engine on Hive, Tez needs to be upgraded to >= `0.10.1` to contain the fix [Tez-4248](https://issues.apache.org/jira/browse/TEZ-4248).
```

Comment:
```suggestion
To use the Tez engine on Hive, Tez needs to be upgraded to >= `0.10.1` which contains a necessary fix [Tez-4248](https://issues.apache.org/jira/browse/TEZ-4248).
```

Reply:
Is the above for Hive >=3? If so, probably worth mentioning it. Does this also mean you need to override the Tez jar files that come with the standard Hive installation?

Reply:
According to [Tez-4100](https://issues.apache.org/jira/browse/TEZ-4100?jql=project%20%3D%20TEZ%20AND%20text%20~%20%223%22), the Tez 0.10.1 release should be working with Hadoop 3.1.3. And for standard Hive installation (default Tez version should be 0.9.2), the Tez jar file needs to be patched. I'll add a comment for that.